disputer/disputer.js

```diff
@@ -0,0 +1,84 @@
+const { Logger } = require("../financial-templates-lib/Logger");
+
+class Disputer {
+  constructor(expiringMultiPartyClient, account) {
+    this.account = account;
+
+    // Expiring multiparty contract to read contract state
+    this.empClient = expiringMultiPartyClient;
+
+    // Instance of the expiring multiparty to perform on-chain liquidations
```

Comment:
nit: typo (this is a dispute bot)
```suggestion
    // Instance of the expiring multiparty to perform on-chain disputes
```